tests/unit/utils/test_utils.py

```diff
@@ -126,6 +127,52 @@ def test_makedirs(repo_dir):
     assert os.path.isdir(path_info.fspath)
 
 
+@pytest.mark.parametrize(
+    "parent",
+    [
+        (os.path.join("path", "to", "")),
+        (os.path.join("path", "to")),
+        (os.path.join("path", "")),
+        (os.path.join("path")),
+    ],
+)
+def test_path_isin_positive(parent):
+    child = os.path.join("path", "to", "folder")
+    assert path_isin(child, parent)
+
+
+def test_path_isin_on_same_path():
+    path = os.path.join("path", "to", "folder")
+    path2 = os.path.join(path, "")
+
+    assert not path_isin(child=path, parent=path)
+    assert not path_isin(child=path, parent=path2)
+    assert not path_isin(child=path2, parent=path)
+    assert not path_isin(child=path2, parent=path2)
+
+
+def test_path_isin_on_different_path():
+    path1 = os.path.join("path", "to", "folder1")
+    path2 = os.path.join("path", "to", "folder")
+
+    assert not path_isin(path1, path2)
+
+
+def test_path_isin_accepts_pathinfo():
```

Comment:
I guess it won't work in Python 3.5 and earlier.

Reply:
Oops, didn't test on `Python3.5`. Fixing ...